### PR TITLE
Hotfix-ms - Fixed font-family for news lead and h3.

### PR DESCRIPTION
**Pull request by mihaelad**

### CODE REVIEW

File: sass/configuration/overrides.scss [hunk 1]
```diff
@@ -78,7 +79,8 @@ h2.pane-title,
   font-family: $font-family-light;
 }
 
-h3.title {
+h3.title,
+.node-ding-news.node-full .field-name-field-ding-news-lead {
```
Comment (guddo) [inline sass/configuration/overrides.scss:83]: I don't this this is only applicable to News CT?

Comment (guddo) [reply sass/configuration/overrides.scss:83]: And why do you want the lead, that should stand out, to be light?

**Comment by guddo** [comment]
Lots of commits for a few changes. I think this should be cleaned up before merging.

File: sass/configuration/overrides.scss (hunk 1 above)
Comment (mihaelad) [reply sass/configuration/overrides.scss]: I added bold font for news and event lead and fix style for bold, strong tags.

**Review by guddo — APPROVED** [review]

**Pull request merged**